bundles/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/NukiBindingConstants.java

```diff
@@ -86,4 +88,12 @@
     // Nuki Binding additional Lock States
     public static final int LOCK_STATES_UNLOCKING_LOCKNGO = 1002;
     public static final int LOCK_STATES_UNLATCHING_LOCKNGO = 1007;
+
+    // Nuki Binding Door States
```

Comment:
These constants don't seem to be used? Can they be removed.

Reply:
They are not required. I added them just like the other LOCK_STATES_.... which are also not used.